src/ts/order.ts

```diff
@@ -61,6 +62,10 @@ export interface Order {
   partiallyFillable: boolean;
 }
 
+export interface OrderCancellation {
+  orderUid: BytesLike;
```

Comment:
Maybe add some doc-comments as this is part of a public-facing API.
```suggestion
/**
 * ...
 */
export interface OrderCancellation {
  /**
   * ...
   */
  orderUid: BytesLike;
```